packages/erxes-ui/src/components/richTextEditor/hooks/useExtensions.ts

```diff
@@ -148,18 +88,7 @@ export default function useExtensions({
       }),
       DivNode,
       SpanMark,
-      CustomLinkExtension.configure({
-        // autolink is generally useful for changing text into links if they
-        // appear to be URLs (like someone types in literally "example.com"),
-        // though it comes with the caveat that if you then *remove* the link
-        // from the text, and then add a space or newline directly after the
-        // text, autolink will turn the text back into a link again. Not ideal,
-        // but probably still overall worth having autolink enabled, and that's
-        // how a lot of other tools behave as well.
-        autolink: true,
-        linkOnPaste: true,
-        openOnClick: false,
-      }),
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider refactoring the commented-out CustomLinkExtension logic.

The large block of commented-out code for CustomLinkExtension might be better refactored or removed to keep the codebase clean.